packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -537,6 +564,20 @@ export class FeatureDevController {
             i18n('AWS.amazonq.featureDev.placeholder.additionalImprovements')
         )
     }
+
+    private async processAutoBuildSetting(setting: boolean, msg: any) {
+        const root = (await this.sessionStorage.getSession(msg.tabID)).config.workspaceRoots[0]
```

Comment:
I think this is used a few places, does it make sense to do this in one place so everything is aligned and we're always pulling the first workspace root?